### PR TITLE
808 score breakdown

### PR DESCRIPTION
**Pull request by Ancentury**

### CODE REVIEW

File: src/views/Results.vue [hunk 1]
```diff
@@ -267,23 +273,90 @@ import Obligations from "@/components/Obligations.vue";
...
 export default class Results extends Vue {
   myResults = this.$store.getters.resultDataSections;
 
+  riskAreaItems() {
+    let items: RiskArea[] = [];
+    let totalNoQuestions = 0;
+    let totalProjectScore = 0;
+    let totalMaxScore = 0;
+    if (this.myResults[1]?.length > 0) {
+      this.myResults[1].forEach((myResult) => {
```
Comment (MrDeshaies) [inline src/views/Results.vue:300]: Is it just me?
```
ERROR in aia-eia-js/src/views/Results.vue(300,34):
300:34 Parameter 'myResult' implicitly has an 'any' type.
```

File: src/views/Results.vue [hunk 2]
```diff
@@ -267,23 +273,90 @@ import Obligations from "@/components/Obligations.vue";
...
+    return items;
+  }
+
+  mitigationItems() {
+    let items: RiskArea[] = [];
+    let totalNoQuestions = 0;
+    let totalProjectScore = 0;
+    let totalMaxScore = 0;
+    if (this.myResults[2]?.length > 0) {
+      this.myResults[2].forEach((myResult) => {
```
Comment (MrDeshaies) [inline src/views/Results.vue:333]: Same error as above: `Parameter 'myResult' implicitly has an 'any' type.`

Comment (Ancentury) [reply src/views/Results.vue]: Fixed

**Comment by MrDeshaies** [comment]
At first glance, it works and looks pretty good! I found two minor issues:
1. In French, the headers Risk Area, No Of Questions, Project Score, etc., and the final total row RAW IMPACT SCORE and MITIGATION SCORE are not translated.
2. From the result page, loading a new json doesn't refresh the scores in the tables. This is how I achieved the screenshot below, showing a "code d'incidence brute" (raw impact score) of 47, but in the table it is shown as 4.
image

**Comment by MrDeshaies** [comment]
One more thing: within the Mitigation section, the table header "Risk Area" should be changed to "Mitigation Area". Thanks!
image

**Comment by Ancentury** [comment]
Good catch. I will update.
1. For the translations. Do you use any specified translations or do I just google it? 
2. I will fix Loading a new json doesn't refresh issue as soon as possible.
3. For the Mitigation section, I just create words similar to Risk Area :) as there is no table instruction. Do you have any specified translations too?
Thank you for finding these.

**Comment by MrDeshaies** [comment]
> 1. For the translations. Do you use any specified translations or do I just google it?
> 3. For the Mitigation section, I just create words similar to Risk Area :) as there is no table instruction. Do you have any specified translations too?

Tables 3 and 4 on [this page](https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/innovations-gouvernementales-numeriques/utilisation-responsable-ai/evaluation-incidence-algorithmique.html) can provide the translations. 
* Risk Area: _Secteur de risque_
* No. of questions: _Nombre de questions_
* Project Score: _Cote du projet_
* Maximum Score: _Cote maximale_
* Mitigation Area: _Secteur d’atténuation_
* RAW IMPACT SCORE: COTE D'INCIDENCE BRUTE
* MITIGATION SCORE: COTE D'ATTÉNUATION

Let me know if you need other ones, or create placeholders in fr.json and I can populate them.

**Comment by Ancentury** [comment]
Thank you for providing the translations. The project is updated. Please check again.

**Review by MrDeshaies — APPROVED** [review]

**Comment by MrDeshaies** [comment]
This PR will need a little more work. The tables always come out empty in deployed versions of the code (like here: https://canada-ca.github.io/aia-eia-js/). I added some details in #808

**Comment by MrDeshaies** [comment]
Fixed with #899

**Pull request merged**